project/src/Behaviors/GSTArrive.gd

```diff
@@ -4,12 +4,17 @@ extends GSTSteeringBehavior
 # The calculation will attempt to arrive with zero remaining velocity.
 
 
+# The target whose location the agent will be steered to arrive at
 var target: GSTAgentLocation
+# The distance from the target for the agent to be considered successfully arrived
```

Comment:
Here, the variable name is clearer than the docstring. I would say something like: "If the agent is at a lower distance from the target than this value, it reached the target."